nixos/modules/services/home-automation/wyoming/openwakeword.nix

```diff
@@ -127,15 +116,34 @@ in
         DynamicUser = true;
         User = "wyoming-openwakeword";
         # https://github.com/home-assistant/addons/blob/master/openwakeword/rootfs/etc/s6-overlay/s6-rc.d/openwakeword/run
-        ExecStart = concatStringsSep " " [
-          "${cfg.package}/bin/wyoming-openwakeword"
-          "--uri ${cfg.uri}"
-          (concatMapStringsSep " " (model: "--preload-model ${model}") cfg.preloadModels)
-          (concatMapStringsSep " " (dir: "--custom-model-dir ${toString dir}") cfg.customModelsDirectories)
-          "--threshold ${cfg.threshold}"
-          "--trigger-level ${cfg.triggerLevel}"
-          "${cfg.extraArgs}"
-        ];
+        ExecStart = escapeSystemdExecArgs (
+          [
+            (lib.getExe cfg.package)
+            "--uri"
+            cfg.uri
+            "--threshold"
+            cfg.threshold
+            "--trigger-level"
+            cfg.triggerLevel
+          ]
+          ++ concatMap (
+            model:
+            [
+              "--preload-model"
+              model
+            ]
+              cfg.preloadModels
+          )
+          ++ concatMap (
+            dir:
+            [
+              "--custom-model-dir"
+              (toString dir)
+            ]
+              cfg.customModelsDirectories
+          )
```

Comment:
Similar issue here, parenthesis appear to be on the wrong line... was this tested? :disappointed:

Reply:
I can feel your frustration. As maintainers, especially the active ones that merge hundreds of PRs, we dont have a perfect hit rate, if you want to try to keep up with the sheer mass of changes, you cant always test everything.
I know it breaks my heart everytime i make a mistake. If you have time to make a PR, im sure he will help you merge it very fast :-)
Thank you for taking the time to report.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/376381. Sorry!